src/main/kotlin/com/cjbooms/fabrikt/model/SourceApi.kt

```diff
@@ -39,10 +39,22 @@ data class SourceApi(
         validateSchemaObjects(openApi3).let {
             if (it.isNotEmpty()) throw ParameterException("Invalid models or api file:\n${it.joinToString("\n\t")}")
         }
-        allSchemas = openApi3.schemas.entries.map { it.key to it.value }
-            .plus(openApi3.parameters.entries.map { it.key to it.value.schema })
-            .plus(openApi3.responses.entries.flatMap { it.value.contentMediaTypes.entries.map { content -> it.key to content.value.schema } })
-            .map { (key, schema) -> SchemaInfo(key, schema) }
+        val globalSchemas = openApi3.schemas.entries.map { it.key to it.value }
+        val globalParameters = openApi3.parameters.entries.map { it.key to it.value.schema }
```

Comment:
**question (non-blocking):** Why did you choose to create explicit variables here instead of continuing with `.plus`  For readibility purpose?

Reply:
Exactly.

Reply:
Thanks for the explanation. I think it looked better before but for readability purposes this is definitely better.